pages/faq.md

```diff
@@ -0,0 +1,182 @@
+---
+slug: faq
```

Comment:
Is this FAQ for the entire site or just AWX?

Reply:
It is for the entire site.  I attempted to communicate the ask for this before and I think it got lost in the shuffle.  The suggestion was to have a stand alone FAQ page, of which I have created here.